bundles/org.eclipse.swt/Eclipse SWT/common/org/eclipse/swt/internal/DPIUtil.java

```diff
@@ -394,6 +392,10 @@ public static void setMonitorSpecificScaling(boolean activate) {
 	System.setProperty(SWT_AUTOSCALE_UPDATE_ON_RUNTIME, Boolean.toString(activate));
 }
 
+public static void setAutoScaleValue(String autoScaleValue) {
```

Comment:
We should add a JavaDoc that states that this method is only supposed to be called in tests and may lead to unintended effects if called in production.